index.bs

```diff
@@ -12334,6 +12335,8 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
    <dd><p>|cookie spec|["<code>expiry</code>"] if it exists, otherwise leave unset to
     indicate that this is a session cookie.
 
+    Note: The cookie&apos;s expiry value might be limited by the remote end in accordance with the [=Cookie Lifetime Limits=].
```

Comment:
Just curious, I don't think we have `&apos;`  in other parts of the spec. Is this better/intentional?